### DISCUSSION TITLE
Latest rope support official yolox==0.3.0?

### DISCUSSION THREAD

**Discussion by leelongc** [mikel-brostrom/yolo_tracking#1214] [Q&A]
I noticed that the previous issues mentioned that this rope only supports SG and ultralytics.

**Comment by mikel-brostrom** [comment]
Not true. yolox 0.3.0 also works :smile:

**Reply by leelongc** [reply]
I use official yolox 0.2.0 and yolox 0.3.0 model for test, which works. But the detect model has no output. The output information is (no detection) in cmd. So i print the variable 'pred' in  yolox.py, which is None. Meanwhile, i also test model from yolox 0.1.0re, ByteTrack and OC-SORT, which can print correct output.
